src/app/sb-admin/components/pages/sb-user/sbuser.component.ts

```diff
@@ -141,15 +106,34 @@ export class SbUserComponent implements OnInit {
     });
   }
 
-    editUser(user: OrganizationsUsersList) {
-        this.dialogService.open(AddEditUserComponent, {
-            data: user,
-            header: this.i18nextPipe.transform('USER_EDIT'),
-            width: '30%',
-            height: 'auto'
-        });
+  loadUserList(event: any) {
+    let filters = this.filteredValue;
+    Object.keys(filters).forEach(key => {
```

Comment:
To enhance the code - we need to utilize the lodash library from the next PR.

Reply:
Ok sir